src/clusterfuzz/_internal/bot/tasks/utasks/fuzz_task.py

```diff
@@ -99,12 +99,6 @@ class FuzzErrorCode:
     ['success', 'testcase_file_paths', 'fuzzer_metadata'])
 
 
-def has_standard_build():
-  if environment.platform() == 'FUCHSIA':
```

Comment:
Why is this being removed and not showing up somewhere else? Is fuchsia deprecated?